src/sqlfluff/rules/L045.py

```diff
@@ -3,10 +3,8 @@
 
 from sqlfluff.core.rules.base import BaseRule, LintResult, RuleContext
 from sqlfluff.core.rules.analysis.select_crawler import Query, SelectCrawler
-from sqlfluff.core.rules.doc_decorators import document_fix_compatible
 
 
-@document_fix_compatible
```

Comment:
Why have you stopped making this fix compatible?

Reply:
It never was "fix compatible" -- if you look at the code, it never returns any `LintFix` objects. So this was always wrong. (It's not related to the rest of the PR, just a thing that I noticed when working on it.) 😛